test/autoprefixer.test.ts

```diff
@@ -370,6 +381,9 @@ it('transition on vendor specific rule', () => {
 it('ignore prefix in vendor at rules', () => {
   check('at-rules')
 })
+it('ignore content property', () => {
+  check('content')
```

Comment:
```suggestion
  let input = read('content')
  let result = postcss([prefixer('scope')]).process(input)
  expect(result.css).toEqual(input)
```

It will allow to avoid the same content in both files.